src/Command/EventMap.php

```diff
@@ -93,7 +111,8 @@ public function mapToBus(CommandBus $bus): array
     {
         $list = [];
         foreach ($this->map as $event => $commands) {
-            $list[$event] = $this->listener($bus, $commands);
+            $extraProperties = $this->staticProperties[$event] ?? [];
+            $list[$event] = $this->listener($bus, $commands, $extraProperties);
```

Comment:
@NerdyHouse Since `$extraProperties` is an array dereferenced from `$this->staticProperties`, and `listener` is a protected function of this class (so it has access to `$this->staticProperties`) I propose that we do not pass it along as part of `$this->listener`'s arguments list.

Instead, we can access `$this->staticProperties[$event]` inside the closure returned from the `listener` method.  The closure gets a copy of the event to use as in index.